src/components/Tooltip/component.stories.tsx

```diff
@@ -50,6 +50,10 @@ const Label = styled.div`
 `;
 
 const PlacementContainer = styled.div`
+  display: flex;
+  flex-direction: row;
```

Comment:
I think the default is already ‘row’ when we declare ‘flex’ 🤔

Reply:
Good catch, I have fixed. Please re-approve when you can. 😄